### PR TITLE
Remove event mpm when using prefork, worker or itk

### PR DESCRIPTION
**Pull request by tuxmea**
fixes https://tickets.puppetlabs.com/browse/MODULES-8903

### CODE REVIEW

File: manifests/mpm.pp [hunk 1]
```diff
@@ -101,11 +101,21 @@
 
       if $mpm == 'prefork' {
```
Comment (ekohl) [inline manifests/mpm.pp]:
```suggestion
      if $mpm in ['prefork', 'worker'] {
```

Comment (tuxmea) [reply manifests/mpm.pp]: AFAIK: prefork must disable event in Debian 9, whereas worker must disable event in Debian 8.
But I might be wrong. We had this issue at a customer today.

Comment (tuxmea) [reply manifests/mpm.pp]: @ekohl can you please review?

**Comment by lionce** [comment]
Screenshot 2019-06-06 at 12 19 10

File: manifests/mpm/disable_mpm_event.pp [hunk 2]
```diff
@@ -0,0 +1,8 @@
+class apache::mpm::disable_mpm_event {
+  @exec { '/usr/sbin/a2dismod mpm_event':
```
Comment (ekohl) [inline manifests/mpm/disable_mpm_event.pp]: Why a virtual resource? Couldn't you just include the class only when it's needed? You can include multiple times

How about a name and the specify the command explicitly? This makes it easier if some distro decides to change it somehow.

**Comment by sheenaajay** [comment]
Thanks for submitting the PR @tuxmea .

**Pull request merged**